index.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const knex = require('knex');
+
+const knexConfig = require('./knexfile.js');
+
+const server = express();
+
+server.use(express.json());
+
+// connect to the database
+const db = knex(knexConfig.development);
+
+server.get('/', (req, res) => {
+  res.send('api working');
+});
+
+
+server.get('/api/cohorts', (req, res) => {
+  db('cohorts')
+    .then(cohorts => {
+      res.status(200).json(cohorts);
+    })
+    .catch(err => res.status(500).json(err));
+});
+
+server.get('/api/cohorts/:id', (req, res) => {
+  db('cohorts')
+    .where({ id: req.params.id })
+    .then(cohort => {
+      if (cohort) {
+        res.status(200).json(cohort);
+      } else {
+        res.status(404).json({ message: 'cohort not found' });
+      }
+    });
+});
+
+server.get('/api/cohorts/:id/students', (req, res) => {
+    let {id} = req.params;
+
+    db('students')
+      .where({ cohort_id: id })
+      .then(student => {
+        if (student) {
+          res.status(200).json(student);
+        } else {
+          res.status(404).json({ message: 'students not found' });
+        }
+      });
+  });
+
+
+server.post('/api/cohorts', (req, res) => {
+  
+  db('cohorts')
+    .insert(req.body)
+    .then(ids => {
+      db('cohorts')
+        .where({ id: ids[0] })
+        .then(cohort => {
+          res.status(201).json(cohort);
+        });
```

Comment:
You have no `catch` for this `then`; while you wouldn't think there'd be an error here, there could be (and I'm not sure if we'd see it in practice or not... but it's usually better to be safe than sorry).